data/json/requirements/toolsets.json

```diff
@@ -24,7 +24,13 @@
     "id": "sewing_standard",
     "type": "requirement",
     "//": "Crafting or repair of fabric items",
-    "tools": [ [ [ "sewing_kit", 1 ], [ "tailors_kit", 1 ], [ "needle_bone", 1 ], [ "needle_wood", 1 ] ] ]
+    "tools": [ [ [ "sewing_kit", 1 ], [ "tailors_kit", 1 ], [ "needle_bone", 1 ], [ "needle_wood", 1 ], [ "sewing_machine", 1 ] ] ]
+  },
+  {
+    "id": "knitting_standard",
```

Comment:
You don't use that anywhere because you defined sewing quality instead.
You should either use the quality or the `using`, not both for the same thing.
Qualities are simpler, while `using`s are easier to extend (for example, if you ever wanted to add an electric knitter that uses power, qualities would not work).